documentation/adoc/appendix_metrics.adoc

```diff
@@ -0,0 +1,131 @@
+[appendix]
+== Metrics
+
+This section describes how to deploy a Prometheus server for scraping metrics from the Kafka cluster and showing them using a Grafana dashboard. The resources provided are examples to show how Kafka metrics can be stored in Prometheus: They are not a recommended configuration, and further support should be available from the Prometheus and Grafana communities.
+
+When using `minikube` or `minishift` for your Apache Kafka deployment adding Prometheus and Grafana servers, it would be better to increase the default amount of memory available to the virtual machine (i.e. not using the default 2 GB of RAM but 4 GB instead).
+
+Information on how to increase the default amount of memory can be found in the following section <<_installing_kubernetes_and_openshift_cluster>>
+
+=== Deploying on OpenShift
+
+==== Prometheus
+
+The Prometheus server configuration uses a service discovery feature in order to discover the pods in the cluster from which it gets metrics.
+In order to have this feature working, it's necessary to provide access to the API server to get the pod list to the service account used for running the Prometheus service pod (the service account `prometheus-server` is used).
+
+[source]
+export NAMESPACE=[namespace]
+oc login -u system:admin
+oc create sa prometheus-server
+oc adm policy add-cluster-role-to-user cluster-reader system:serviceaccount:${NAMESPACE}:prometheus-server
+oc login -u developer
+
+where `[namespace]` is the namespace/project where the Apache Kafka cluster was deployed.
+
+After that, it's possible to deploy the Prometheus server in the cluster :
+
+. Create the https://github.com/strimzi/strimzi/blob/master/metrics/examples/prometheus/openshift-template.yaml[provided "prometheus" template] by running
++
+[source]
+oc create -f metrics/examples/prometheus/openshift-template.yaml
```

Comment:
@scholzj Can this mislead the user? The user can think that he don't need download .yaml file before and he just needs to execute the command "oc create -f <file_or_dir_path>" where <file_or_dir_path> will be not a path to file on some environment but the URL.

Reply:
It should actually work ... the `;oc` tool or `kubectl` would automatically download the file from the URL and use it. So the user can just copy paste the command.

Reply:
After discussion, we decided to use url for the raw file because direct link resulted in an error "error converting YAML to JSON: yaml: line 321: mapping values are not allowed in this context"